app/upw/templates/privacy.njk

```diff
@@ -4,18 +4,16 @@
 {% from "common/templates/components/question/macro.njk" import renderQuestion %}
 {% from "common/templates/components/widgets/macro.njk" import widgets %}
 
-{% set backLink = 'task-list' %}
-
 {% block pageTitle %}
 {{ pageTitle }}
 {% endblock %}
 
 {% block header %}
+{%- set usingBrowserBack = true -%}
```

Comment:
We're having to set a toggle as FormWizard is automagically setting the `backLink` variable, this will use the `previousUrl` instead